clients/python/tests/import_failure.py

```diff
@@ -6,8 +6,6 @@
 os.environ["RUST_LOG"] = "foo=bar"
 
 try:
-    import tensorzero  # noqa
```

Comment:
This line is the whole point of the test :)

Reply:
wait wtf I didnt' mean to remove it
I think precommit did

Reply:
sorry!

Reply:
confirmed pre-commit did it. Bad pre-commit!